source/import-font-and-images-assets/index.html.md.erb

```diff
@@ -12,28 +12,30 @@ To use the font and image assets from GOV.UK Frontend, you can either:
 
 ## Serve the assets from the GOV.UK Frontend assets folder - recommended
 
-Set up your routing so requests for files in `<YOUR-SITE-URL>/assets` are served from `/node_modules/govuk-frontend/dist/govuk/assets`.
+<%= warning_text('The file paths on this page point to asset folders that were updated in GOV.UK Frontend v5.10.0') %>
+
+Set up your routing so requests for files in `<YOUR-SITE-URL>/assets/rebrand` are served from `/node_modules/govuk-frontend/dist/govuk/assets/rebrand/rebrand`.
 
 For example if you're using [express.js](https://expressjs.com/), add the following to your `app.js` file:
 
 ```javascript
 const path = require('path');
-app.use('/assets', express.static(path.join(__dirname, '/node_modules/govuk-frontend/dist/govuk/assets')))
+app.use('/assets/rebrand', express.static(path.join(__dirname, '/node_modules/govuk-frontend/dist/govuk/assets/rebrand')))
```

Comment:
If we go with the suggestion to keep the `assets` path rather than `assets/rebrand`, revert this change.